services/business/istio_validations.go

```diff
@@ -79,9 +80,9 @@ func (in *IstioValidationsService) GetIstioObjectValidations(namespace string, o
 	case "gateways":
 		// Validations on Gateways are not yet in place
 	case "virtualservices":
-		if vs, err = in.k8s.GetVirtualService(namespace, object); err == nil {
+		if vss, err = in.k8s.GetVirtualServices(namespace, ""); err == nil {
```

Comment:
Did you check this with the UI, too ?
I mean, this endpoint was intended to return just the validation of a single object, and now it will return all validations of the namespace.
I understand that we need all, but wondering if for this endpoint, we should cut rest of the validation and return only the specific for the virtualservice requested.
(But yes, we need all of them for the group validation).

Reply:
Yes, I did. And that is something I want to comment with you. I see that ACEditor is taking the whole object validators even it prints one single object. Should ACEditor accept only one ObjectValidator?
Also, in case that in one yaml we have more than one Istio Object, the currect ACEditor would place the marks into each istio object properly?
I did a test letting ACEditor use just one object validator and worked correctly.

anyway, no problem on cutting all the leftovers. Actually, saying leftovers, looks like is the best option (for data transfer reduction sake).

Reply:
> Should ACEditor accept only one ObjectValidator?

I don't think it is necessary to refactor this. Just knowing that the backend send validator for a single object is enough.

In theory it should work well, at least with the usecases we implemented for previous checkers.

Reply:
Alright. I already add the change.
btw, UI needs a really small tweak. I already made the PR: kiali/kiali-ui#504.